R/global_sessions_config.R

```diff
@@ -29,6 +29,8 @@
 #' allow user to do certain actions (such as viewing charts and tables) without signing in,
 #' and only require users to sign in if they want to save data to your database.
 #' @param sentry_dsn either \code{NULL}, the default, or your Sentry project DSN.
+#' @param cookie_expiration the number of days before a User's cookie expires.
```

Comment:
don't capitalize User's

Reply:
Also in the docs.  mention "This argument is passed to the "expires" option in js-cookie: \url{https://github.com/js-cookie/js-cookie#expires}"